src/AdoNet/Orleans.Clustering.AdoNet/AdoNetHostingExtensions.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using Microsoft.Extensions.DependencyInjection;
+using Orleans.Messaging;
+using Orleans.Runtime.Membership;
+using Orleans.Runtime.MembershipService;
+using Orleans.AdoNet.Configuration;
+using Orleans.AdoNet.Options;
+
+namespace Orleans.Hosting
+{
+    public static class AdoNetHostingExtensions
+    {
+        /// <summary>
+        /// Configure SiloHostBuilder with SqlMembership
```

Comment:
Worth noting there is a string "sql" used rather copiously too. Maybe something like
> Configure SiloHostBuilder using ADO.NET membership.

I wonder also, since it seems to be in .NET API docs that whole sentenced starting with a capital letter and full stop is used if these should be fixed too? If not rejected, I could volunteer a PR for that for ADO.NET too as I was thinking to do something over the weekend. :)

Reply:
Good catch. Fixed.